src/pocketmine/network/mcpe/protocol/StartGamePacket.php

```diff
@@ -83,8 +83,8 @@ class StartGamePacket extends DataPacket{
 	public $hasAchievementsDisabled = true;
 	/** @var int */
 	public $time = -1;
-	/** @var bool */
-	public $eduMode = false;
+	/** @var int */
+	public $eduEditionOffer = 0;
```

Comment:
I ask you to make some changes to Player about this. (In Player you can find sending StartGamePacket and do something with it, otherwise there will be extra crashdumps :P)

Reply:
Thanks